OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -247,7 +247,7 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 					if (!order.Queued)
 						self.CancelActivity();
 
-					self.SetTargetLine(target, Color.Yellow);
+					self.ShowTargetLines();
```

Comment:
Shouldn't this (and the cases in the files below) be done after the QueueActivity?

Reply:
I just replaced the lines that go SetTargetLine to ShowTargetLines so the changes are easier to see. The target line drawing happens for some ticks so that it will display correct activity queue after one tick. On the other hand, you are right about this.